### ISSUE TITLE
Rework prepublishing checklist

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#34456]
## Issue Description
We need to review our prepublishing checklist and add more detail and links so that any content author who needs to publish a page knows the steps to follow. 

We should have the accessibility and design system teams review the document to see if we need to add any additional information and to provide documentation we can link to for certain items. 

## Tasks
- [x]  Rework checklist (Randi)
- [x]  Share with accessibility (Randi)
- [x] Incorporate accessibility feedback (Randi)
- [x] Compare against Platform's standards to determine if anything should be added
- [x] Share with design system team (Randi)
- [x] Incorporate design feedback (Randi)
- [x] Review final draft (Beth)

## Acceptance Criteria
- [x] Prepublishing checklist is updated for all content authors to understand

**Comment by RLHecht** [comment]
@bethpottsVADEPO I finished reworking this and separated the list into different sections. I also added an item to confirm that the page meetings Platform's experience standards with a link to their chart on the platform website. Here's the updated file: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/checklists/prepublishing-checklist.md

**Comment by GnatalieH** [comment]
> @GnatalieH We wanted to share our updated pre-publishing checklist with you to review. This is what we use before we publish any of our static pages on the site. We wanted you to take a look at it to see if there's anything you think still needs to be included from a design system standpoint. Thanks!

I'll take a look today, @RLHecht. Thanks for sharing!

**Comment by RLHecht** [comment]
Thanks for bringing this up @GnatalieH. We had never thought about adding the file size to the end of non_HTML links. We brought up the topic in our Sitewide Content sync today and will discuss it further and let you know where we land. @bethpottsVADEPO Do you want to review the checklist one more time before we close this ticket?

**Comment by RLHecht** [comment]
Work on the checklist is complete. We've been discussing the PDF file size in a separate slack thread. This ticket can be closed.
